Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -379,6 +408,14 @@ void MainWindow::ShowAboutDialog()
   about->show();
 }
 
+void MainWindow::ShowHotkeyDialog()
+{
+  m_hotkey_window->ChangeMappingType(MappingWindow::Type::MAPPING_HOTKEYS);
+  m_hotkey_window->show();
```

Comment:
Well wouldn't that be a waste of resources? If you open the window multiple times it will have to get resconstructed from scratch every single time.